apps/backend/src/iteration-helpers/repo-processor.ts

```diff
@@ -28,7 +28,8 @@ export class RepoProcessor extends ItemProcessor<Repo> {
       .from(repos)
       .orderBy(desc(repos.added_at))
       .offset(skip)
-      .leftJoin(projects, eq(projects.repoId, repos.id));
+      .leftJoin(projects, eq(projects.repoId, repos.id))
+      .groupBy(repos.id);
```

Comment:
Needed because repos can be linked to multiple projects, we want each repo to be processed only once!